### ISSUE TITLE
Aliases do not resolve for tests

### ISSUE THREAD

**Issue by samdemaeyer** [oklas/react-app-rewire-alias#17]
Following the docs, the tests will not resolve the aliases.
In my project, but also in the example version of the codebase.

When pulling the latest version, and trying to run the tests, the aliases are not resolved, unfortunately.

<img width="653" alt="Screenshot 2021-02-09 at 10 16 56" src="https://user-images.githubusercontent.com/7160913/107349514-37f4fe80-6ac0-11eb-80c4-e0631d2d5f7e.png">

**Comment by oklas** [comment]
I updated dependencies - this does not reproduces. Try `rm -rf node modules && yarn`. Check current commit 397125a.

**Comment by samdemaeyer** [comment]
Hi @oklas, I have synced up my fork without luck. 
To illustrate the problem, I added a [GitHub Action to my fork](https://github.com/samdemaeyer/react-app-rewire-alias/pull/1), there the dependencies are being fetched from scratch. You can see that the tests don't pass because the alias is not resolved.
```bash
Cannot find module 'above-root-js/AboveRootJs' from 'src/App.tsx'
```
You can have a look at the tests here 👇 
https://github.com/samdemaeyer/react-app-rewire-alias/pull/1/checks?check_run_id=1864761315

Also .. in your latest commit (https://github.com/oklas/react-app-rewire-alias/commit/397125a827cc6ae509ee5782c424dfe447e1910f), `example/main/yarn.lock` is out of sync, you changed `example/main/package.json` but not `example/main/yarn.lock`
If I run `yarn` inside `examples/main`, `yarn.lock` is still altered. (not related to the issue though)

<img width="792" alt="Screenshot 2021-02-09 at 16 53 58" src="https://user-images.githubusercontent.com/7160913/107398331-6d690e80-6af7-11eb-80e7-1669cbd3c654.png">

**Comment by oklas** [comment]
Alias support for test jest added.

Now aliases is resolved but dangerous mode requires imports to be configured #18. 

Available from 1.0.0
